lib/tools/getCampaign.tsx

```diff
@@ -6,7 +6,7 @@ import { tool } from "ai";
 const getCampaign = (question: string) =>
   tool({
     description: `IMPORTANT: Always call this tool for ANY question related to the following topics:
-    1. Artists
+    1. Artists(except the case related to analytics and management!!!)
```

Comment:
I think you could remove this point overall. 🚮 
- get campaign - return the status of an active campaign